concourse/scripts/pipecleaner_test.go

```diff
@@ -77,8 +77,8 @@ var _ = Describe("PipeCleaner", func() {
 			})
 
 			It("should warn about the non-portable compare, and exit as though fatal", func() {
```

Comment:
Rather than adjusting this test, it should be removed.

It was testing that a warning would fail in the context of "with --fatal-warnings", but it's not a warning, so the entire test is meaningless now.